### DISCUSSION TITLE
Quickapps flatpaks

### DISCUSSION THREAD

**Discussion by sitolam** [prasanthrangan/hyprdots#1662] [Q&A]
How to use flatpaks with the `quickapps.sh` script?

**Comment by sitolam** [comment]
I fixed it by copying the `.desktop` file from `~/.local/share/flatpak/exports/share/applications` to `/usr/share/applications` but I don't know if that is a smart way of doing it.

**Comment by dieBakterie** [comment]
I think you need to add the path for flatpaks, but i'm not sure. You should just add them in the `~/.config/waybar/modules/wbar.jsonc`and you should be fine. you could just add the path to the quickapps.sh script to achieve the same.

**Reply by sitolam** [reply]
How do I need to add it in the `quickapps.sh` script?

**Reply by dieBakterie** [reply]
Or you could add this path(`~/.local/share/flatpak/exports/share/applications`) to the quickapps script but not sure if that works tbh

**Reply by dieBakterie** [reply]
In theory a restart/reload of the environment would be enough, flatpak isn't in the path until that. Hmm

**Reply by sitolam** [reply]
I tried adding it like this in the `quickapps.sh` script:
![image](https://github.com/prasanthrangan/hyprdots/assets/79202140/af33eb57-9536-47cc-b6ed-8256de2868cd)
but that doesn't work.

**Reply by dieBakterie** [reply]
yeah you need to add it not like this, 'cause the second matches and so the third option never gets used or the first matches and the third and second doesn't get used. A reload should *copy* the .desktop entries from flatpak as they aren't in the path yet and so not in your apps list so rofi don't know them yk?

**Reply by dieBakterie** [reply]
The first doesn't exist for me but the second directory and so it uses /usr/share/applications as appDir and never the third as above. Let me try something real quick.

**Reply by dieBakterie** [reply]
Armcord for example is for me in `/var/lib/flatpak/app/xyz.armcord.ArmCord/x86_64/stable/3e93c41df2ad3bef47a7567531cff414f74dd84f93337dbf4332b70c25b3802a/files/bin/armcord/armcord.desktop` as you can see it's a little difficult to support flatpaks, easiest way i to copy/link the desktop to the for you matching app directory i guess?

**Reply by sitolam** [reply]
I guess but isn't there a symlink in `~/.local/share/flatpak/exports/share/applications` of all flatpak applications? With me it's like that.

**Reply by dieBakterie** [reply]
Nope for me there is only db and repo no exports folder

**Reply by dieBakterie** [reply]
And its flatpak run xyz.armcord.Armcord for example and not just `    ~  vesktop`

**Reply by dieBakterie** [reply]
I'll try and install through the .extra script the flatpaks and see if then i've this folder
